src/basics.php

```diff
@@ -15,9 +15,9 @@
 use Cake\Core\Configure;
 use Cake\Error\Debugger;
 
-/**
- * Basic defines for timing functions.
- */
+    /**
+     * Basic defines for timing functions.
+     */
     define('SECOND', 1);
     define('MINUTE', 60);
     define('HOUR', 3600);
```

Comment:
Its a bit odd that these defines are indented at all. At least the doc block is consistent now.